apps/docs/src/app/platform/component-docs/platform-approval-flow/platform-approval-flow-examples/platform-approval-flow-example.component.ts

```diff
@@ -79,6 +92,14 @@ export class PlatformApprovalFlowExampleComponent implements OnDestroy {
         console.log('Node click handler', node);
     }
 
+    afterNodeAdd(node: ApprovalNode): void {
+        const meta = this._approvalFlow._graphMetadata[node.id];
```

Comment:
Let's define a public method to retrieve node metadata by giving node id instead of touching `_graphMetadata` directly cause underscore means that it is private one

Reply:
Done